docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: global
+title: Running Facebook Presto with Alluxio
+nickname: Presto
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Facebook Presto](https://prestodb.io/) with Alluxio, so
+that you can easily use presto to query Hive tables in Alluxio's tiered storage.
+
+# Prerequisites
+
+The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must greater then
+1.8. Alluxio cluster should also be set up in accordance to these guides for either
```

Comment:
The os environment must use the java 1.8. You can see it in this link https://github.com/prestodb/presto.  Alluxio do not need to recompiled with 1.8. But Alluxio need to update guava's verison to 18.0 in client.